justfile

```diff
@@ -14,6 +13,22 @@ docker-build crate tag=default_docker_tag:
 install-cli:
   cargo install --path ./crates/astria-cli --locked
 
+# version is a celestia-app release. Example: v1.9.0
+# operatingSystem is Linux or Darwin
+# machineHardwareName is arm64 or x86_64
+celestia_default_appd_dst := "."
+get-celestia-appd version operatingSystem machineHardwareName dst=celestia_default_appd_dst:
```

Comment:
No, because we don't install it. `cargo install foo` puts foo in a well known spot (`~/.cargo/bin`) that is expexted to be in `PATH`. I don't really know where to put this binary because I don't have knowledge of the users PATH.

Either way, we can remove this entirely when moving to a stateful set.